srb2kart.sh

```diff
@@ -5,7 +5,14 @@ for i in {0..9}; do
 done
 
 export PATH="/app/utils/gamescope/bin:$PATH"
-export SRB2WADDIR=/app/extra
+
+if [[ -z "$SRB2WADDIR" ]]; then
+	export SRB2WADDIR=/app/extra
+fi
```

Comment:
Would there be any instance where `$SRB2WADDIR` is defined outside the Flatpak runtime?

Reply:
yes, the most clear example is when using custom 3d models. you need to change mdls.dat and add some files to the mdsl folder.